ui/app/src/Views.elm

```diff
@@ -16,10 +19,44 @@ import Views.NavBar.Views exposing (navBar)
 view : Model -> Html Msg
 view model =
     div []
-        [ navBar model.route
-        , div [ class "container pb-4" ]
-            [ currentView model ]
+        [ renderLink "https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css"
```

Comment:
@fabxc Yes, for sure. See *outdated* [comment](https://github.com/prometheus/alertmanager/pull/836#discussion_r119600791) above, but if you don't mind we would do it in a separate PR.